### PR TITLE
Subviews: own Sprite surface on copy

### LINKED ISSUE #282: [subviews] Color picker controls are broken
In every game (except PST), the color picker controls are still broken:  There's only just one color, ~or sometimes none at all, such as in BG~:

bg_colors

### PR DESCRIPTION
**Pull request by MarcelHB**
Closes #282. Color pickers look like they still have some script issues sometimes, but we address this later.

### CODE REVIEW

File: gemrb/plugins/SDLVideo/SDLSurfaceSprite2D.cpp [hunk 1]
```diff
@@ -59,31 +59,29 @@ SDLSurfaceSprite2D::SDLSurfaceSprite2D(const SDLSurfaceSprite2D &obj)
 	: Sprite2D(obj)
```
[the remark this answers is not in the document]
Comment (MarcelHB) [reply gemrb/plugins/SDLVideo/SDLSurfaceSprite2D.cpp]: Yes, I checked that.

**Comment by MarcelHB** [comment]
Thanks, I will incorporate your feedback tomorrow.

**Comment by MarcelHB** [comment]
@bradallred Please check again.

**Comment by bradallred** [comment]
perfect thanks!

**Pull request merged**